k8s/eks/testground_install.sh

```diff
@@ -0,0 +1,193 @@
+echo "========================"
+echo ""
+echo "Now configuring storage.."
+echo ""
+echo "Let's check utility..."
+jq_check >> ./log/$start-log/cluster.log
+echo "Creating EFS (Elastic File System)..."
+aws_create_file_system >> ./log/$start-log/cluster.log
+echo "EFS created with ID : $efs_fs_id "
+echo ""
+echo "Now extracting subnet group ID..."
+aws_get_subnet_id >> ./log/$start-log/cluster.log
+echo "Subnet group ID: $subnet_id"
+echo ""
+# echo "Now extracting CIDR block..."
+# aws_get_subnet_cidr_block >> ./log/$start-log/cluster.log
+# echo "CIDR block is: $subnet_cidr_block"
+# echo ""
+# echo "Creating security group for EFS..."
+# aws_create_efs_sg >> ./log/$start-log/cluster.log
+# echo "EFS security group created with ID: $efs_sg_id"
+# echo ""
+# echo "Now authorising subnet CIDR block $subnet_cidr_block to access $efs_sg_id "
+# aws_efs_sg_rule_add >> ./log/$start-log/cluster.log
+aws_get_sg_id >> ./log/$start-log/cluster.log
+echo "Done."
+echo ""
+
+echo "Creating EFS mount point"
+aws_create_efs_mount_point >> ./log/$start-log/cluster.log
+echo "Your EFS mountpoint DNS is: $efs_dns"
+echo ""
+echo "Now creating EFS configmap"
+create_cm_efs >> ./log/$start-log/cluster.log
+echo ""
+echo "Now creating EFS manifest"
+create_efs_manifest >> ./log/$start-log/cluster.log
+
+echo "========================"
+echo ""
+echo ""
+
+echo "Creating EBS volume (Elastic Block Store)"
+aws_create_ebs >> ./log/$start-log/cluster.log
+echo "EBS created with this volume ID: $ebs_volume "
+echo ""
+echo "Now making persistent volume on EKS"
+make_persistent_volume >> ./log/$start-log/cluster.log
+echo "Done."
+echo "========================"
+
+echo "Setting up Redis..."
+helm_redis_add_repo >> ./log/$start-log/cluster.log
+echo "Helm Redis repo added"
+echo "Now proceeding with Helm install"
+helm_infra_install_redis >> ./log/$start-log/cluster.log
+echo "Redis installed"
+echo "========================"
+echo "We will setup the testground daemon now."
+echo ""
+echo ""
+echo "Creating configmap..."
+tg_daemon_config_map >> ./log/$start-log/cluster.log
+echo "Creating service account..."
+tg_daemon_service_account >> ./log/$start-log/cluster.log
+echo "Creating role binding..."
+tg_daemon_role_binding >> ./log/$start-log/cluster.log
+echo "Creating the testground service.."
+tg_daemon_services >> ./log/$start-log/cluster.log
+echo "Creating svc sync service..."
+tg_daemon_svc_sync_service >> ./log/$start-log/cluster.log
+echo "Creating sync service deployment..."
+tg_daemon_sync_service >> ./log/$start-log/cluster.log
+echo "Creating sidecar..."
+tg_daemon_sidecar >> ./log/$start-log/cluster.log
+echo "Creating testground daemon deployment..."
+tg_daemon_deployment >> ./log/$start-log/cluster.log
+
+echo "========================"
+echo ""
+echo ""
+echo "Your cluster is ready to be used."
+echo ""
+log
+echo "Log for this build can be fonud on this path ./log/$start-$CLUSTER_NAME-$CNI_COMBINATION.tar.gz"
```

Comment:
Do you mean having `tee` to output to both `stdout` and the log file? Because that would defeat the purpose of readability of the output, just like it is for the kops-based script.
We want our output to be nice and clean, and if someone wants to tail the logs, they are free to do so from a different terminal window, as noted in the README.

Reply:
All I'm saying is that right now stderr is only visible on the console and if you share the tar.gz package with someone, they will never see the stderr because it doesn't contain it.

Reply:
both stdout and stderr have been redirected to the log file.